pom.xml

```diff
@@ -6,7 +6,7 @@
     <version>1.645</version>
   </parent>
   <artifactId>collabnet</artifactId>
-  <version>2.0.6-SNAPSHOT</version>
+  <version>2.0.6</version>
```

Comment:
AFAIK, this is done automatically during the release process

Reply:
OK